std_msgs/Header header
float64 ship_latitude             # ship_latitude reading
float64 ship_latitude_variance    # ship_latitude_variance reading

